README.md

```diff
@@ -37,19 +37,19 @@ For library usage check [Docs.rs](https://docs.rs/rimage/latest/rimage/)
 
 ### List of supported Codecs
 
-| Image Format | Decoder       | Encoder                 |
-| ------------ | ------------- | ----------------------- |
-| bmp          | zune-bmp      | -                       |
-| jpeg         | zune-jpeg     | mozjpeg or jpeg-encoder |
-| png          | zune-png      | oxipng or zune-png      |
-| avif         | libavif       | ravif                   |
-| webp         | webp          | webp                    |
-| ppm          | zune-ppm      | zune-ppm                |
-| qoi          | zune-qoi      | zune-qoi                |
-| farbfeld     | zune-farbfeld | zune-farbfeld           |
-| psd          | zune-psd      | -                       |
-| jpeg-xl      | jxl-oxide     | zune-jpegxl             |
-| hdr          | zune-hdr      | zune-hdr                |
+| Image Format | Decoder       | Encoder                 | NOTE                                                 |
+| ------------ | ------------- | ----------------------- | ---------------------------------------------------- |
+| bmp          | zune-bmp      | -                       | Input only                                           |
```

Comment:
Does `-` not implies that encoding (output) not supported?

Reply:
I think it should implies that rimage do not support it, maybe we could use symbles like "X", "⛌", "☒", "×" or "⛔" to avoid any possible of misunderstanding.

Reply:
In my opinion, `Encoder` aims to write the pic to disk and `Decoder` is used to read the pic to bytes

Reply:
Ok, you can use X symbol